src/components/Home.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react'
+import { getList } from '../api/pokemons.api'
+import Post from '../components/Post'
+
+function Home() {
+    const [posts, setPosts] = useState<Post[]>([])
+  
+    interface Post {
+      name: string;
+      url: string;
+    }
+  
+    useEffect(() => {
+      getList(0, 9).then(values => setPosts(values.data.results))
+    })
```

Comment:
Same here with dependencies array
```ts
useEffect(() => {
  getList(0, 9).then(values => setPosts(values.data.results))
}, [])
```
This will prevent infinite request calls.